torch/utils/_sympy/value_ranges.py

```diff
@@ -70,16 +70,41 @@ def vr_is_expr(vr: ValueRanges[_T]) -> TypeGuard[ValueRanges[sympy.Expr]]:
     return not vr.is_bool
 
 
+ExprIn = Union[int, float, sympy.Expr]
```

Comment:
Seems like these should all be typed as typing_extensions.TypeAlias

Reply:
Can't? It's 3.10 only

Reply:
oh, maybe typing_extensions will work?

Reply:
@ezyang typing_extensions backports it to 3.8. `typing.TypeAlias`  is 3.10+ only

Reply:
This doesn't seem necessary here? And it seems to be deprecated too?

Reply:
It's really only necessary with string annotations, wasn't aware it was deprecated.